### PR TITLE
Fixed UI test problems introduced after the accessibility identifier …

### PR DESCRIPTION
**Pull request by stefanceriu**
…changes

### CODE REVIEW

**Review by Velin92 — CHANGES_REQUESTED** [review]
Maybe would be better to remove the line `"parallelizable" : true` from the UI tests .xctestplan to avoid multiple UI tests to run in parallel, which seems to break the testing in Xcode. (At least for me locally).

Also the following screenshot test seems to be failing:
```
class OnboardingUITests: XCTestCase {
    func testInitialStateComponents() {
        let app = Application.launch(.onboarding)
        app.assertScreenshot(.onboarding)
    }
```

**Comment by stefanceriu** [comment]
> Maybe would be better to remove the line `"parallelizable" : true` from the UI tests .xctestplan to avoid multiple UI tests to run in parallel, which seems to break the testing in Xcode. (At least for me locally).

I think we should, doesn't work properly for me either

**Comment by stefanceriu** [comment]
> Also the following screenshot test seems to be failing:

Seems we have broken our `elementFallbackLanguage` in the UITests refactoring

Screenshot 2023-02-16 at 08 56 47

File: ElementX/Sources/UITests/UITestsAppCoordinator.swift [hunk 1]
```diff
@@ -34,13 +34,13 @@ class UITestsAppCoordinator: AppCoordinatorProtocol {
     }
     
     func start() {
+        Bundle.elementFallbackLanguage = "en"
```
Comment (pixlwave) [inline ElementX/Sources/UITests/UITestsAppCoordinator.swift:37]: Whoops sorry, didn't think about that!

File: UITests/Sources/Application.swift [hunk 2]
```diff
@@ -24,7 +24,6 @@ struct Application {
             "IS_RUNNING_UI_TESTS": "1",
             "UI_TESTS_SCREEN": identifier.rawValue
         ]
-        Bundle.elementFallbackLanguage = "en"
```
Comment (pixlwave) [inline UITests/Sources/Application.swift:27]: I think this is about setting the fallback language in the tests runner? Not sure it really matters with our strings though.

**Review by pixlwave — APPROVED** [review]
LGTM

File: UITests/Sources/Application.swift (hunk 2 above)
Comment (pixlwave) [reply UITests/Sources/Application.swift:27]: Changed my mind, we probably do need to keep this to match a label value in the tests to what is shown in the app when running in German.

**Review by pixlwave — APPROVED** [review]

**Pull request merged**